meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -524,6 +524,7 @@ LMP_FLASHLAYOUT_BOARD_NAME:stm32mp15-eval = "stm32mp157c-ev1"
 LMP_BOOT_FIRMWARE_FILES:stm32mp15-eval = "arm-trusted-firmware/tf-a-stm32mp157c-ev1-emmc.stm32 fip/fip-stm32mp157c-ev1-optee.bin"
 MACHINE_ESSENTIAL_EXTRA_RDEPENDS:append:stm32mp15-eval = " kernel kernel-devicetree u-boot-default-script"
 MACHINE_ESSENTIAL_EXTRA_RDEPENDS:remove:sota:stm32mp15-eval = "kernel kernel-devicetree u-boot-default-script"
+KERNEL_DEVICETREE:stm32mp15-disco = "stm32mp157c-ev1-scmi.dtb"
```

Comment:
look like this is for `stm32mp15-eval` an not `stm32mp15-disco`

Reply:
fixed, thanks